docs/uenv-prgenv-gnu.md

```diff
@@ -101,6 +96,9 @@ There are three ways to access the software provided by prgenv-gnu, once it has
     # method 2: enable modules after the uenv has started
     uenv start prgenv-gnu/24.7
     uenv view modules
+
+    # with v6 (TODO or 7?) of uenv:
+    uenv start --view=modules prgenv-gnu/24.7
```

Comment:
@bcumming could you confirm how the versioning works? Is it the 7.0.0 version of `uenv2` that removes the `uenv view` command?

Reply:
Version 7.

Reply:
Thanks @bcumming. Updated to only mention v7.